logger.go

```diff
@@ -0,0 +1,50 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/evalphobia/logrus_sentry"
+	"github.com/sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+// F is a map used to log custom fields.
+type F map[string]interface{}
```

Comment:
Where is it being used?

Reply:
It's used in example below, this example is on readme.

```go
package main
import "github.com/cyberlabsai/logger"
func main() {
    // Note that the F is just a easy access to a map.
    logger.WithFields(log.F{
		"user_id": 1,
		"func":    "customMessage",
		"message": "my custom message",
	}).Info("When i log with custom fields")
}
```